lib/matplotlib/tests/test_figure.py

```diff
@@ -66,6 +66,31 @@ def test_align_labels():
     fig.align_labels()
 
 
+@image_comparison(['figure_align_titles'], extensions=['png', 'svg'],
```

Comment:
I wonder if we need both png and svg here - is the outcome likely to be affected by file format?

Reply:
Removed svg test